src/main/java/org/mitre/synthea/engine/Transition.java

```diff
@@ -99,6 +111,90 @@ public String follow(Person person, long time) {
     }
   }
 
+  /**
+   * A transition that is based on the type of care that will follow. Transitions may be made to
+   * three states, "ambulatory", "emergency" and "emergency". The probability that a person will
+   * follow a particular transition path is based on the time in the simulation (telemedicine is
+   * more likely during and after the COVID-19 pandemic) and the type of insurance the person has.
+   */
+  public static class TypeOfCareTransition extends Transition {
+    private String ambulatory;
+    private String telemedicine;
+    private String emergency;
+    private TelemedicineConfig config;
+
+    /**
+     * Creates a new telemedicine config essentially from the JSON in GMF. This also reads in the
+     * telemedicine_config.json files which contains the different distributions to be used for
+     * transitions.
+     * @param options The states to transition to
+     */
+    public TypeOfCareTransition(TypeOfCareTransitionOptions options) {
+      this.ambulatory = options.ambulatory;
```

Comment:
I suppose it's a personal preference. The code in `follow` doesn't have to worry about reaching into a `TypeOfCareTransitionOptions` object. That said, it would be a simple matter of calling a getter and the code where that getter would be called is pretty clear.